docs/platform/what-is-tizen/overview.md

```diff
@@ -77,6 +77,13 @@ Compiler Toolchain
    - Neural network model quantizer.
    - Various profiles and test scripts to evaluate performance on target.
 
+### MediaVision
+MediaVision provides task based inference API; classification, detection, and pose landmark detection.
+It also provides easy ways for a user:
```

Comment:
When a colon introduces a list, the text that precedes the colon must be able to stand alone as a complete sentence.
Ref: https://developers.google.com/style/colons#introductory-phrase-preceding-colon
```suggestion
It also provides an easy way for a user to do the following:
```